docs/how-to/nat-configuration.md

```diff
@@ -1,32 +1,132 @@
 1. Locate your routers port forwarding section.
 1. Enter the IP address of your IPFS node.
-1. Set traffic to go through port `4001`.
+1. Set traffic to go through an outside port. 
+   :::tip
+   If you are unsure of what port to use, `4001` is recommended.
+   :::
 1. Reboot your IPFS node for the changes to take effect. Make sure to reboot the entire machine, not just the IPFS daemon.
 
-Assuming you're not trying to expose your daemon's API to the public internet, opening port `4001`/`tcp` should be sufficient.
+Now that you've opened a port from the internet to your internal Kubo node, update your Kubo configuration to set `Swarm.AppendAnnounce` as a list of addresses that other IPFS nodes will try to contact you at.
+
+#### Update the Kubo Configuration
+
+In this step, you will update your Kubo configuration to set `Swarm.AppendAnnounce` as a list of addresses that other IPFS nodes will try to contact you at. This list lets other nodes on the network that the port forward you created in the previous step know that your node exists, and that these are the addresses at which you can be contacted at. To update your configuration:
+
+1. Open your Kubo configuration file. 
+
+   :::tip
+   The default location for the config file is `~/.ipfs/config`. If you have set `$IPFS_PATH`, you can find your config file at `$IPFS_PATH/config`.
+   :::
+   
+1. Find the entry for `AppendAnnounce` in `Addresses` or create an entry for `AppendAnnounce` in `Addresses`:
+
+   ```json
+    "Addresses": {
+        "Swarm": [
+        "/ip4/0.0.0.0/tcp/4001",
+        "/ip6/::/tcp/4001",
+        "/ip4/0.0.0.0/udp/4001/quic",
+        "/ip6/::/udp/4001/quic"
+        ],
+        "Announce": [],
+        "AppendAnnounce": [],
+        "NoAnnounce": [],
+        "API": "/ip4/127.0.0.1/tcp/5001",
+        "Gateway": "/ip4/127.0.0.1/tcp/8080"
+    },
+    ``` 
+
+1. Update `AppendAnnounce`, where `<public-ip>` is your public IP address and `<port>` is the port number set in the previous step: 
+
+   ```json
+   "AppendAnnounce": [
+     "/ip4/<public-ip>/tcp/<port>",
+     "/ip4/<public-ip>/udp/<port>/quic",
+     "/ip4/<public-ip>udp/<port>/quic-v1",
+     "/ip4/<public-ip>/udp/<port>/quic-v1/webtransport",
+    ],
+   ```
+
+   For example, if your public IP address is `1.2.3.4` and `12345` is the port you've chosen, `AppendAnnounce` would look like:
+
+   ```json
+   "AppendAnnounce": [
+     "/ip4/1.2.3.4/tcp/12345",
+     "/ip4/1.2.3.4/udp/12345/quic",
+     "/ip4/1.2.3.4/udp/12345/quic-v1",
+     "/ip4/1.2.3.4/udp/12345/quic-v1/webtransport",
+    ],
+   ```
+
+Now that you've updated your Kubo configuration, restart Kubo so that your node is reachable, and check that the outside port is open.
+
+#### Restart your Kubo node
+
+1. Reboot your IPFS node for the changes to take effect. Make sure to reboot the entire machine, not just the IPFS daemon. Once Kubo restarts, your node should be reachable. 
+1. Check that the outside port is open.
```

Comment:
Consider adding an additional step to this, one that describes how the user would check that the outside port is open.

Reply:
I agree. However, I don't think this should block adding the rest of this in. I'd vote push as is, and then we can update that step early next week